src/Compiler/NodeVisitor/Behavior/ComponentsNodeVisitorBehavior.php

```diff
@@ -33,6 +34,11 @@ private function findComponentByName(array $components, string $componentName):
             if (count($componentNameParts) === 0) {
                 return $component;
             }
+            $componentType = $component->getType();
+            if ($componentType instanceof ObjectType && $componentType->isInstanceOf('Nette\Application\UI\Multiplier')->yes()) {
+                // TODO: How to determine component type?
```

Comment:
maybe we should collect them as wildcard name subcomponents, I don't use Multiplier so I don't know how it works but from test case I assume we could find all return types of components in Multiplier constructor and use `*` as their name, then we check `*` here in  Visitor.

I'm just not sure why there is not `multiplied` used in latte. How Nette solves the problem which component should be rendered if there are more than one controls in Multiplier?

Reply:
ou, it's not array, it's callable factory O:-) Then it is only this one type returned by `createComponentMultiplied`

Reply:
Yes Multiplier is just simple object that creates sub-components on demand. But we need to know what type these subcomponents have.

Reply:
We know it because factory has some return type, in this case it is Form